lego-webapp/components/JoblistingItem/JoblistingItem.module.css

```diff
@@ -49,14 +49,27 @@ img.companyLogo {
   margin-bottom: var(--spacing-sm);
 }
 
-.deadline {
+.infoLeft {
   display: flex;
-  align-items: center;
-  justify-content: flex-end;
+  flex-direction: column;
+  justify-content: space-between;
```

Comment:
A lot of this properties can be set in the Flex component (e.g. flex-direction). Also `display: flex` is not needed.